templates/repo/release/list.tmpl

```diff
@@ -39,6 +39,9 @@
 								</h3>
 								<div class="download gt-df gt-ac">
 									{{if $.Permission.CanRead $.UnitTypeCode}}
+										{{if .CreatedUnix}}
+											<span class="time gt-mr-3 gt-mono muted">{{svg "octicon-clock" 16 "gt-mr-2"}}{{TimeSinceUnix .CreatedUnix $.locale}}</span>
```

Comment:
Wait.

1. What does "time gt-mono muted" mean here? I guess many (all) of them are misused.
2. Take a look at #21440
    * The relative time should follow the best practice at the moment.

Reply:
- `gt-mono` seems unnessary, it's only meant for the commit hash to render in monospace font
- `muted` is needed for the link color

Reply:
But it isn't a like. It is a `span`.

Reply:
Fixed in my latest commit.